Sources/SwiftDocC/Model/Rendering/Content/RenderBlockContent.swift

```diff
@@ -76,7 +76,10 @@ public enum RenderBlockContent: Equatable {
     
     /// A video with an optional caption.
     case video(Video)
-
+    
+    /// A horizontal rule.
```

Comment:
If I understand correctly the "horizontal rule" terminology is somewhat specific to the HTML element and "thematic break" is a more abstract term that describe the use of this element. 

```suggestion
    /// An authored thematic break between block elements.
```